pkg/cryptoauthlib/include/atca_params.h

```diff
@@ -53,15 +54,15 @@ extern "C" {
 #define ATCA_RX_RETRIES          (20)
 #endif
 #ifndef ATCA_DEVTYPE
-#define ATCA_DEVTYPE            (ATECC508A)
+#define ATCA_DEVTYPE            (ATECC608A)
```

Comment:
This is good **but** I just realized that this conflicts with the SAML11 boards that contain a ATECC508A (and don't expose it). Could you please include explicit parameters to the board.h file of the SAMLs please